components/crypto/src/rand.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! Utilities for cryptographically strong random number generation.
+
+use openssl::{error::ErrorStack, rand};
+
+/// Fill buffer with cryptographically strong pseudo-random bytes.
+pub fn rand_bytes(buf: &mut [u8]) -> Result<(), ErrorStack> {
+    rand::rand_bytes(buf)
+}
+
+/// Return a random u64.
+pub fn rand_u64() -> Result<u64, ErrorStack> {
```

Comment:
It's a bit weird that get random number can fail? Can we fallback to system rand if this call failed?

Reply:
+1, can we just unwrap it

Reply:
In the context of cryptographic random number generation, RNG may fails due to lack of entropy.
And FIPS 140 has requirements to RNG, the default system RNG may not meet the requirements. 
So we can not simply unwrap the error.